src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -251,16 +255,16 @@ public void setType(Material type, boolean applyPhysics)
 	@Deprecated
 	public boolean setTypeId(int type)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		Material old = material;
+		this.material = Material.getMaterial(type);
```

Comment:
```suggestion
		setType(material);
```
BlockMock.setType(Material) and BlockMock.setTypeId(int) should actually do the same thing, but it would not do, when I would use chest ID.